Bug-Debian: http://bugs.debian.org/594893
#
#
# patch "pidgin/libpurple/network.c"
#  from [8c70d2a63b2c464b174ff8cc768e43a6bff9c4cb]
#    to [040edad982c3770eb34822415ad9341218f66fa8]
#
============================================================
Index: pidgin/libpurple/network.c
===================================================================
--- pidgin.orig/libpurple/network.c
+++ pidgin/libpurple/network.c
@@ -1077,12 +1077,10 @@ purple_network_remove_port_mapping(gint
 
 	if (protocol) {
 		purple_network_upnp_mapping_remove(&port, protocol, NULL);
-		g_hash_table_remove(upnp_port_mappings, protocol);
 	} else {
 		protocol = g_hash_table_lookup(nat_pmp_port_mappings, &port);
 		if (protocol) {
 			purple_network_nat_pmp_mapping_remove(&port, protocol, NULL);
-			g_hash_table_remove(nat_pmp_port_mappings, protocol);
 		}
 	}
 }
